js/Source/Request/Request.PollForUpdate.js

```diff
@@ -63,10 +63,10 @@ Request.PollForUpdate = new Class({
     var data = this.request.response.json;
     // if the server provides an 'updated_at' timestamp,
     // store as an attribute
-    if (data.status == 'update'){
+    if (typeof data !== 'undefined' && data.status == 'update'){
```

Comment:
data was undefined, and data.status raises an error when data is undefined, right?